fapi-client/src/main/scala/com/gu/facia/api/utils/CardStyle.scala

```diff
@@ -0,0 +1,101 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.Content
+import com.gu.facia.api.utils.ContentApiUtils._
+import com.gu.facia.client.models.MetaDataCommonFields
+
+object CardStyle {
+
+  def isExternalLink(content: Content, metaDataFields: MetaDataCommonFields): Boolean = (for {
+    snapType <- metaDataFields.snapType
+    href <- metaDataFields.href.orElse(content.safeFields.get("href"))
+  } yield snapType == "link" && ExternalLinks.external(href)) getOrElse false
+
+  def apply(content: Content, trailMetaData: MetaDataCommonFields): CardStyle = {
+    if (isExternalLink(content, trailMetaData)) {
+      ExternalLink
+    } else if (content.tags.exists(_.id == "news/series/hsbc-files")) {
+      SpecialReport
+    } else if (content.isLiveBlog) {
+      if (content.isLive) {
+        LiveBlog
+      } else {
+        DeadBlog
+      }
+    } else if (content.isPodcast) {
+      Podcast
+    } else if (content.isMedia) {
+      Media
+    } else if (content.isEditorial) {
+      Editorial
+    } else if (content.isComment) {
+      Comment
+    } else if (content.isAnalysis) {
+      Analysis
+    } else if (content.isReview) {
+      Review
+    } else if (content.isLetters) {
+      Letters
+    } else if (content.isFeature) {
+      Feature
+    } else {
+      Default
+    }
+  }
+}
+
+sealed trait CardStyle {
```

Comment:
Would this be simpler as just a string?

  object CardStyles {
    val specialReport = "special-report"
    val toneString = "live"
    // ... etc ...
  }

Even if not, you can take out the explicit type annotations for each of these. Also, why are they defs instead of vals? Using def for the abstract member is good (so it can be implemented either way) but the implementations don't need to be lazy or recalculated every time.